src/main/java/uk/gov/hmcts/darts/audio/component/impl/OutboundFileProcessorImpl.java

```diff
@@ -90,13 +90,14 @@ public List<AudioFileInfo> processAudioForPlaybacks(Map<MediaEntity, Path> media
         List<AudioFileInfo> concatenatedAndMergedAudioFileInfos = new ArrayList<>();
         for (ChannelAudio audioFileInfoList :  concatenationsList) {
             AudioFileInfo mergedAudio = merge(audioFileInfoList.getAudioFiles());
+            OffsetDateTime concatStartTime = mergedAudio.getStartTime().atOffset(ZoneOffset.UTC);
```

Comment:
Should this variable be called `mergedAudioStartTime`?
I don't think the atOffset bit is necessary as the Duration.between method inside trimToPeriod will use the first arg start instant anyway, so I actually think these 2 variables can probably be dropped.